### ISSUE TITLE
Add license info to the remote dataset documentation

### ISSUE THREAD

**Issue by meghanrjones** [GenericMappingTools/remote-datasets#27]
This is a request that Kurt Schwehr made in response to our twitter post announcing the datasets to add license info so that folks know if the datasets are okay for a particular use.

https://twitter.com/gmt_dev/status/1493343171601915908

**Comment by PaulWessel** [comment]
Seems reasonable.  I think the USGS are public domain [so mag and images], got permission to share wdmam in GMT, but we would need to dig out others.  Do you want to take a crack at a list?

**Comment by meghanrjones** [comment]
It would be nice to have this information more convenient for users, but I expect it would take time to compile if any are using non-standard licenses. Right now, I would prefer to keep my attention on some of the bigger tasks (e.g., speeding up the test suite, getting the bundle fixed with GEOS and the wrappers) that keep getting pushed aside by a stream of small maintenance tasks. Since we do provide the references for the users to find this information, could we back-burner this until those issues are addressed?

**Comment by PaulWessel** [comment]
yes of course.  I don't think we have any license problems that needs immediate addressing.

**Comment by meghanrjones** [comment]
Great, thanks
